x-pack/plugins/actions/server/lib/create_action_event_log_record_object.ts

```diff
@@ -66,6 +77,12 @@ export function createActionEventLogRecordObject(params: CreateActionEventLogRec
       })),
       ...(spaceId ? { space_ids: [spaceId] } : {}),
       ...(task ? { task: { scheduled: task.scheduled, schedule_delay: task.scheduleDelay } } : {}),
+      action: {
```

Comment:
nit: since `name` and `actionExecutionId` are both optional, we should avoid writing out the action block if both are undefined.

Reply:
Resolved in this commit 5104f14d5b742547cfeff3c233988eb90ae1f484